regression/goto-analyzer/sensitivity-test-constants-pointer-to-two-value-struct/test.desc

```diff
@@ -9,12 +9,12 @@ sensitivity_test_constants_pointer_to_two_value_struct.c
 ^\[main.assertion.4\] .* p->a==1: Unknown$
 ^\[main.assertion.5\] .* p->b==2.0: Unknown$
 ^\[main.assertion.6\] .* p->b==1.0: Unknown$
-\[main\.assertion\.7\] .* comp_p==&x\.a: Unknown$
+\[main\.assertion\.7\] .* comp_p==&x\.a: Success$
```

Comment:
I would expect any improvements in this test should be at least matched in the constant-struct version of this test?

Reply:
@thk123 Which one is that?

Reply:
`sensitivity-test-constants-pointer-to-constants-struct` - specifically assert 7 is not checked in this file:
https://github.com/diffblue/cbmc/blob/goto-analyzer-develop/regression/goto-analyzer/sensitivity-test-constants-pointer-to-constants-struct/test.desc